JamGame/Assets/Scripts/Level/Shop/Employee/CardView.cs

```diff
@@ -106,13 +106,18 @@ public void TryHireEmployee()
                 return;
             }
 
-            if (controller.TryBuyEmployee(EmployeeConfig).Success)
+            Result result = controller.TryBuyEmployee(EmployeeConfig);
+            if (result.Success)
             {
                 coinIcon.sprite = hiredIconSprite;
                 hireCostLabel.gameObject.SetActive(false);
                 hireButton.interactable = false;
                 hired = true;
             }
+            else
+            {
+                Debug.Log("Cannot hire employee: " + result.Error);
```

Comment:
Add `TODO` and create an issue to show pop-up here as well as in case of `Not enough money`